.drone.yml

```diff
@@ -1,297 +1,109 @@
-    image: quay.io/ukhomeofficedigital/kd:v1.16.0
-    environment:
-      - ENVIRONMENT=wcs-dev
-      - VERSION=build-${DRONE_BUILD_NUMBER}
-      - HOCS_INFO_SERVICE_DATA_VERSION=latest
-      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
-    secrets:
-      - hocs_info_service_dev_wcs
-    commands:
-      - cd kube-hocs-info-service
-      - ./deploy.sh
-    when:
-      branch: master
-      event: [push, tag]
-
-  deployment:
-    image: quay.io/ukhomeofficedigital/kd:v1.16.0
-    environment:
-      - ENVIRONMENT=${DRONE_DEPLOY_TO}
-      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
-    secrets:
-      - hocs_info_service_dev_cs
-      - hocs_info_service_demo_cs
-      - hocs_info_service_dev_wcs
-      - hocs_info_service_demo_wcs
-    commands:
-      - cd kube-hocs-info-service
-      - ./deploy.sh
-    when:
-      event: deployment
-      environment: [cs-dev, cs-demo, wcs-dev, wcs-demo]
+environment:
+  KEYCLOAK_SERVER_ROOT: http://keycloak:8080
+  KEYCLOAK_USER: admin
+  KEYCLOAK_PASSWORD: password1
 
-  deploy-to-qa-cs:
-    image: quay.io/ukhomeofficedigital/kd:v1.16.0
-    environment:
-      - ENVIRONMENT=cs-qa
-      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
-    secrets:
-      - hocs_info_service_qa_cs
-    commands:
-      - source version.txt
-      - echo $VERSION
-      - cd kube-hocs-info-service
-      - ./deploy.sh
-    when:
-      event: deployment
-      environment: cs-qa
-
-  deploy-to-qa-wcs:
-    image: quay.io/ukhomeofficedigital/kd:v1.16.0
+services:
+  - name: keycloak
+    image: jboss/keycloak
     environment:
-      - ENVIRONMENT=wcs-qa
-      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
-    secrets:
-      - hocs_info_service_qa_wcs
-    commands:
-      - source version.txt
-      - echo $VERSION
-      - cd kube-hocs-info-service
-      - ./deploy.sh
-    when:
-      event: deployment
-      environment: wcs-qa
-
-  deploy-to-hocs-qax:
-    image: quay.io/ukhomeofficedigital/kd:v1.16.0
+      DB_VENDOR: h2
+  - name: postgres
+    image: quay.io/ukhomeofficedigital/postgres-alpine
     environment:
-      - ENVIRONMENT=hocs-qax
-      - KUBE_SERVER=https://kube-api-notprod.notprod.acp.homeoffice.gov.uk
-    secrets:
-      - hocs_info_service_qax
-    commands:
-      - source version.txt
-      - echo $VERSION
-      - cd kube-hocs-info-service
-      - ./deploy.sh
-    when:
-      event: deployment
-      environment: hocs-qax
+      POSTGRES_USER: root
+      POSTGRES_PASSWORD: dev
 
-  deploy-to-prod-cs:
-    image: quay.io/ukhomeofficedigital/kd:v1.16.0
-    environment:
-      - ENVIRONMENT=cs-prod
-      - KUBE_SERVER=https://kube-api-prod.prod.acp.homeoffice.gov.uk
-    secrets:
-      - hocs_info_service_prod_cs
+steps:
+  - name: wait for keycloak
+    image: quay.io/ukhomeofficedigital/hocs-docker-tools
+    commands:
+      - >
+        until
+          $(curl \
```

Comment:
You shouldn't need the `\`. `>` should replace all the `\n` with spaces.

Reply:
As proven through Drone removing these does not work.